content/guides/user-guides/delfin/dashboard/_index.md

```diff
@@ -0,0 +1,368 @@
+---
+title: SRM using SODA Dashboard
```

Comment:
Can we use Storage Infrastructure Management (SIM)

Reply:
Fixed. made this change.